test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -557,53 +554,12 @@ static RepairOption previewOption(RandomSource rs, Cluster.Node coordinator, Str
 
     private static RepairOption repairOption(RandomSource rs, Cluster.Node coordinator, String ks, Gen<List<String>> tablesGen, Gen<RepairType> repairTypeGen, Gen<PreviewType> previewTypeGen, Gen<RepairParallelism> repairParallelismGen)
     {
-        RepairType type = repairTypeGen.next(rs);
```

Comment:
rather than having different tests randomize repair stuff, i pulled this out into `RepairGenerators` so this logic could be reused